### ISSUE TITLE
Rebuilding SDL Library failed. No Libs/DLLs generated. No Error-Message!

### ISSUE THREAD

**Issue by zzador** [beefytech/Beef#211]
Tried to fix the little bug in the packaged SDL-Lib myself (see: https://github.com/beefytech/Beef/issues/210) but when hitting "Build solution" it says:

`Compiling...
Beef compilation time: 0.02s
Executing Command: CopyToDependents("$(ProjectDir)/dist/*.dll")
Total build time: 0.05s`

However the "build\Release_Win64\SDL2"-subdirectory is empty and there are no LIBs or DLLs that were copied to "dist/" (Have deleted the LIBs and DLLs there to see if new ones would be created).

Don't know how to rebuild SDL at the moment. :(

PS.: Using Beef 0.42.3

**Comment by bfiete** [comment]
I'm not understanding your bug.

Is there an error when you actually run the program? Is the program running but you don't see the effect of your changes? Do you even HAVE a program or are you just directly loading the SDL project file into an empty workspace and then expecting to see some sort of results on your file system that you're not seeing?

**Comment by zzador** [comment]
I have opened the project of the packaged SDL-Library in the subdir "BeefLibs/SDL". Then I wanted to build the library. The build process returned successfully but a look into the "build" directory shows 2 more directories "corlib" & "SDL". The "corlib" directory is filled with obj-files and a "_corlib.lib" file but the "SDL" directory is empty. I first thought that the CopyToDependents would delete (therefore move) the files in the build directory, but the targetdirectory "dist" remains empty too (after deletion of the prebuild LIBs and DLLs that are already there).

Long story short: Building the SDL-Lib creates corlib-library but **no** SDL-library -> sdl2.lib & DLLs

**Comment by bfiete** [comment]
Beef libraries are not buildable targets unto themselves. The compilation model is explained a bit here: https://www.beeflang.org/docs/language-guide/basics/

The CopyToDependents command copies the SDL2 redistributable DLLs to an target project's directory. It has nothing to do with the output of the Beef SDL2 wrapper library.

You do not have to manually "rebuild SDL" - you made the source change and that's all you need to do. You're done. There's no bug here.

**Comment by zzador** [comment]
Oh sorry, by bad. Didn't know that. Guess I'll close this issue then.
